parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -291,16 +304,18 @@ pub mod pallet {
 			Ok(())
 		}
 
-		// Cross check to make sure ExecutionHeader not fall behind FinalizedHeader too much, if
-		// that happens just return error so to pause processing FinalizedHeader until
-		// ExecutionHeader catch up
+		/// Cross check to make sure the ExecutionHeader does not fall too far behind
+		/// FinalizedHeader updates. If that happens just return an error so to pause processing
+		/// FinalizedHeader until ExecutionHeader processing has caught up.
```

Comment:
We don't really have a `FinalizedHeader`type, so I felt this would read better:

```suggestion
		/// Cross check to make sure that execution header import does not fall too far behind
		/// finalised beacon header import. If that happens just return an error and pause processing
		/// until execution header processing has caught up.
```